packages/in17n/package.json

```diff
@@ -0,0 +1,72 @@
+{
+	"name": "@sapphire/plugin-in17n",
+	"version": "1.0.0",
+	"description": "> TODO: description",
+	"author": "@sapphire",
+	"maintainers": [
+		"QuantumlyTangled <yo@quantumlytangled.com>"
+	],
+	"contributors": [
+		"QuantumlyTangled <yo@quantumlytangled.com>"
+	],
+	"license": "BSD-3-Clause",
+	"main": "dist/index.js",
```

Comment:
Please base the package.json from the other plugins, this is missing a bunch of stuff, including ESM support.